resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/engines/ApacheHttpClient43Engine.java

```diff
@@ -269,7 +269,11 @@ public Response invoke(Invocation inv)
       }
       catch (Exception e)
       {
-         throw new ProcessingException(Messages.MESSAGES.unableToInvokeRequest(), e);
+         ProcessingException processingException = new ProcessingException(
+                 Messages.MESSAGES.unableToInvokeRequest(), e);
+         // record error details in server.log
```

Comment:
Sorry to keep harping on this. In the testsuite, the message goes to server.log because the test is running in wildfly, but, typically, the message would get logged on the client side.

Reply:
fixed